.github/workflows/quality.yml

```
@@ -21,3 +21,33 @@ jobs:
       - run: npm ci
       - run: npm run lint
       - run: npm test
+  
+  e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.9)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)